### ISSUE TITLE
Are you using only ImageNet's train set and val set?

### ISSUE THREAD

**Issue by sdsdsz** [MC-E/TransCL#4]
Hello, I would like to ask if you are using only the test set and validation set for training and testing?
